docs/image.md

```diff
@@ -12,6 +12,7 @@ The following defines the relevant fields of the `image` resource spec in more d
 - `builder`: Configuration of the `builder` resource the image builds will use. See more info [Builder Configuration](builders.md).
 - `serviceAccountName`: The Service Account name that will be used for credential lookup.
 - `source`: The source code that will be monitored/built into images. See the [Source Configuration](#source-config) section below.
+- `activeDeadlineSeconds`: Optional configurable max active time that the pod building the image can run for
```

Comment:
We should probably call this buildTimeout or similar. I think it would be easier to teach and use for end users rather than active deadline seconds in the image. It also doesn't make sense at the image spec level since the image resource itself doesn't have a deadline, rather the builds it creates. Thoughts?

Reply:
That makes sense to me regarding the changing the name to `buildTimeout` 👍🏻 

Do you have any ideas on a better place to put it than image spec?

Reply:
Ah, the latter sentence was about the active deadline seconds field itself. If we were to rename the image spec field to build timeout, I think that would be great and solve my concerns :)

Reply:
Updated now 👍🏻

Reply:
Could `activeDeadlineSeconds` go on the `spec.build` to indicate it is a build specific property?

Reply:
Updated @matthewmcnew 👍🏻